com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalUI.cs

```diff
@@ -30,18 +30,7 @@ enum Expandable : uint
 
         protected override void OnMaterialGUI(MaterialEditor materialEditor, MaterialProperty[] props)
         {
-            // always instanced
-            SerializedProperty instancing = materialEditor.serializedObject.FindProperty("m_EnableInstancingVariants");
```

Comment:
This part have disappeard and I don't think it is expected  (to setup instancing to true)

Reply:
ah sorry , see it is setup below

Reply:
I moved it below in SetupCommonDecalMaterialKeywordsAndPass, which is called by the resetkeyword code. That way, we don't need the user to open the inspector to open the flag, and we don't make use of SerializedProperty, which works kind of unreliably for materials